File: 백준/Bronze/10801. 카드게임/카드게임.cc
#include <iostream>
#include <vector>
using namespace std;
int main(void) {
    vector<int> A(10), B(10);
    int cntA = 0, cntB = 0;
    for (int i = 0; i < 10; i++)
        cin >> A[i];
    for (int i = 0; i < 10; i++)
        cin >> B[i];
    for (int i = 0; i < 10; i++) {
        if (A[i] == B[i])
            continue;
        if (A[i] > B[i])
            cntA += 1;
        else
            cntB += 1;
    }
    cout << (cntA == cntB ? "D" : (cntA > cntB ? "A" : "B"));
    return 0;
}

File: 백준/Bronze/10801. 카드게임/카드게임.py
import sys
input=lambda:sys.stdin.readline().rstrip()
cntA,cntB,cntD=0,0,0
A=[*map(int,input().split())]
B=[*map(int,input().split())]
for i in range(10):
    if A[i]==B[i]:
        continue
    else:
        if A[i]>B[i]:
            cntA+=1
        elif A[i]<B[i]:
            cntB+=1
print('D' if cntA==cntB else 'A' if cntA>cntB else 'B')